dev/release/setup-rhel7-rebuilds.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# A script to install dependencies required for release
+# verification Red Hat Enterprise Linux 7 rebuilds in particular
+# on CentOS 7
+
+set -exu
+
+yum -y update
+yum -y groupinstall "Development Tools"
+yum -y install centos-release-scl curl
+releasever="$(rpm --query --file --queryformat '%{VERSION}\n' /etc/system-release)";
+basearch="$(rpm --query --queryformat '%{ARCH}\n' "kernel-$(uname --kernel-release)")";
+cd /etc/pki/rpm-gpg
+curl -O http://springdale.princeton.edu/data/springdale/$releasever/$basearch/os/RPM-GPG-KEY-springdale
+cd /etc/yum.repos.d
+cat << EOF > Springdale-SCL.repo
+[Springdale-SCL]
+name=Springdale - SCL
+baseurl=http://springdale.princeton.edu/data/springdale/SCL/$releasever/$basearch
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-springdale
+EOF
+cd $HOME
```

Comment:
Can we remove this because we can use LLVM 13 from `centos-release-scl`?

Reply:
Ok. Checking this.

Reply:
Without Gandiva, LLVM13 from centos-release-scl is fine. Gandiva requires clang and clang-tools which are not provided by centos-release-scl